aldryn_forms/models.py

```diff
@@ -17,6 +19,7 @@
 from .helpers import is_form_element
 
 
+USES_TREEBEARD = LooseVersion(cms.__version__) >= LooseVersion('3.1')
```

Comment:
AFAIK ``treebeard`` is only present starting from ``cms 3.1`` since that was one of the major goals of that release - to switch to ``treebeard``, and i think that ``cms 3.0`` has no plans to switch to ``treebeard``. So I agree with @czpython this is suitable, though we can rename it to be ``USE_TREEBEARD_ORDERING`` instead of ``USE_TREEBEARD``